recipe/meta.yaml

```diff
@@ -1,23 +1,24 @@
-{% set version = "4.3.0" %}
+{% set version = "4.4.0" %}
 
 package:
   name: jupyter_client
   version: {{ version }}
 
 source:
   fn: jupyter_client-{{ version }}.tar.gz
-  url: https://github.com/jupyter/jupyter_client/archive/{{ version }}.tar.gz
-  md5: 257d9f5429dac4d9511db84d201d3a9e
+  url: https://pypi.io/packages/source/j/jupyter_client/jupyter_client-{{ version }}.tar.gz
+  sha256: c99a52fac2e5b7a3b714e9252ebf72cbf97536d556ae2b5082baccc3e5cd52ee
 
 build:
   number: 0
-  script: python setup.py install
+  script: pip install --no-deps .
```

Comment:
Yes, since it's only the entrypoints that are an issue, conda entrypoints clobbers the ones that don't work and we are okay. Once the pip entrypoints work, these become redundant and can be considered for removal.